### PR TITLE
Combined STI prev

### PR DESCRIPTION
**Pull request by kweiss2**
Adding combined STI prevalence and incidence, assuming that incid.ct and incid.gc are counts

### CODE REVIEW

File: R/mod.prevalence.R [hunk 1]
```diff
@@ -156,6 +156,10 @@ prevalence_msm <- function(dat, at) {
   dat$epi$ir100.uct[at] <- (dat$epi$incid.uct[at] / sum(uCT == 0, na.rm = TRUE)) * 5200
   dat$epi$ir100.ct[at] <- (dat$epi$incid.ct[at]/ sum(rCT == 0 | uCT == 0, na.rm = TRUE)) * 5200
 
+  dat$epi$prev.sti[at] <- sum(rGC == 1 | ugc == 1 | rCT ==1 | uCT == 1, na.rm = TRUE) / dat$epi$num[at]
+  dat$epi$ir100.sti[at] <- ((dat$epi$incid.ct[at] + dat$epi$incid.gc)/ sum(rCT == 0 | uCT == 0 | rGC == 0 | uGC ==0, na.rm = TRUE)) * 5200
```
Comment (smjenness) [inline R/mod.prevalence.R]: Typo, ugc

**Comment by kweiss2** [comment]
The incidence statistic might need to account for dual-infection (i.e. CT + NG - (NG&CT))?

File: R/mod.prevalence.R (hunk 1 above)
Comment (smjenness) [inline R/mod.prevalence.R]: Actually, the denominator for the incidence stats should link the STIs with an `&` not an `|` (my mistake). Can you change for the other incidence stats too?

Comment (kweiss2) [inline R/mod.prevalence.R]: No problem!

**Comment by smjenness** [comment]
No regarding dual-infection. It should be handled all with the STIs chained by ANDs

**Comment by smjenness** [comment]
Looks good, proceed as we discussed earlier...

**Comment by kweiss2** [comment]
The STI incidence statistic doesn't seem to work: 
`Error: missing values and NaN's not allowed if 'na.rm' is FALSE`
Can you see the fix I just tried to add?

**Comment by smjenness** [comment]
No you may need to republish the branch

**Pull request merged**